generators/spa/index.js

```diff
@@ -14,7 +14,8 @@ module.exports = class extends DnnGeneratorBase {
         choices: [
           { name: 'Angular', value: 'Angular' },
           { name: 'ReactJS', value: 'ReactJS' },
-          { name: 'VueJS', value: 'VueJS' }
+          { name: 'Vue 2', value: 'VueJS' },
+          { name: 'Vue 3', value: 'Vue3JS' }
```

Comment:
@alejoroman0605 You ignored my note on this. I asked you to change this to `Vue 3`.  CC: @ismaelTartola

Reply:
@WillStrohl I did not ignore this comment, what I did was change the name, which is what is displayed in the console, but I immediately changed the value as well.

Reply:
@WillStrohl The index.js uses the value to search the template folder, I modified this line: 
let spaPath = spaType === "ReactJS" ? `${this.props.spaType}/${this.props.langType}` : spaType === "Vue 2" ? "VueJS/" : spaType === "Vue 3" ? "Vue3JS/" : `${this.props.spaType}/`;
so it would find the template files, is that okay? Otherwise I would have to rename the VueJS and Vue3JS folder.

Reply:
@alejoroman0605 When I looked at this before your response, the values did not appear to be updated.